x-pack/plugins/security_solution/public/common/components/search_bar/index.tsx

```diff
@@ -479,6 +461,41 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
     dispatch(inputsActions.setSearchBarFilter({ id, filters })),
 });
 
+const useSyncSearchBarUrlParam = () => {
```

Comment:
The same here, this file is getting very large. Could we move this hook outside? together with `useInitSearchBarUrlParams`, maybe in a `public/common/hooks/search_bar/*` folder?